.github/scripts/main/solvers.sh

```diff
@@ -30,9 +30,9 @@ case "$SOLVER" in
 esac
 
 opam update --depexts
-opam switch create $SOLVER ocaml-system || true
+opam switch create "$SOLVER" ocaml-system || true
 opam upgrade --all
-opam install $PKGS
+opam install "$PKGS"
```

Comment:
```suggestion
opam install $PKGS
```

Reply:
Good catch, thanks!
I'm looking at other instances of this bug

Reply:
Should be fixed now